src/Appwrite/Migration/Version/V23.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Appwrite\Migration\Version;
+
+use Appwrite\Migration\Migration;
+use Exception;
+use Throwable;
+
+class V23 extends Migration
```

Comment:
We can remove for now. We can do this later when all is merged in 1.7.x

Reply:
Tests are breaking without it.